### ISSUE TITLE
Adding 2D Sprite

### ISSUE THREAD

**Issue by Haargeeel** [Rajawali/Rajawali#1301]
Hey guys,

I'm trying to add a 2D Picture next to a 3D model. I encountered different problems. In the beginning it looks like this, which is okay:

![screenshot_2014-09-10-14-39-46](https://cloud.githubusercontent.com/assets/4594660/4217478/2f0ff740-38e8-11e4-98b4-74158b04ce2c.png)
The left one is the image and the right one the 3D model.
I want to be able to rotate the models. Rotating on the y-axis is no problem. But when I want to rotate it on the x-axis, it rotates around the bottom but I need it to rotate around the center. If anyone knows how to do that, it would be a huge help.

Because I can't rotate the model on the x-axis, I rotate the camera around the model, but because of reasons, I don't understand the `PointSprite` is not facing the camera the whole time:

![screenshot_2014-09-10-14-40-13](https://cloud.githubusercontent.com/assets/4594660/4217531/c62d71b6-38e8-11e4-9e6a-f015f3c47567.png)
I add the sprite in the `initScene` method like this: 
```
            leftSprite = new PointSprite(leftWidth, leftHeight);
            Material material = new Material();
            material.addTexture(new Texture("leftSprite", leftBitmap));
            material.setColorInfluence(0f);
            leftSprite.setMaterial(material);
            leftSprite.side = -leftWidth / 2;
            leftSprite.top = leftHeight;
            getCurrentScene().addChild(leftSprite);
```

Basically I just want the sprite facing the camera the whole time. 

Does anyone have an idea?

Thanks in advance :)

**Comment by jwoolston** [comment]
Closing this issue as resolved. As for suggestions, post the request to the G+ community.
